src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,43 @@
 
 class Animal {
   // write your code here
+  constructor(health = 100, name) {
+    this.health = health;
+    this.name = name;
+  };
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
   // write your code here
+  constructor(name, hidden = false) {
+    super(100, name);
```

Comment:
please make Herbivore constructor take name and health, instead of hardcoding 100, also you don't have to take/pass hidden property
```suggestion
    super(100, name);
```